builtin/providers/google/resource_compute_instance_template.go

```diff
@@ -134,7 +143,6 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 			"automatic_restart": &schema.Schema{
 				Type:       schema.TypeBool,
 				Optional:   true,
-				Default:    true,
```

Comment:
Have you verified that this works without `automatic_restart` set? Also removing this default could be dangerous to existing configs.